src/patternfly/components/DatePicker/examples/DatePicker.md

```diff
@@ -12,7 +12,12 @@ import './DatePicker.css'
 ### Basic
 ```hbs
 {{#> date-picker date-picker--id="basic"}}
-  {{> form-control controlType="input" input="true" form-control--modifier="pf-m-icon pf-m-calendar" form-control--attribute=(concat 'type="text" value="03/05/2020" id="' date-picker--id '-input" name="' date-picker--id '-input" aria-label="Basic date picker example"')}}
+  {{#> input-group}}
+    {{> form-control controlType="input" input="true" form-control--attribute=(concat 'type="text" value="03/05/2020" id="' date-picker--id '-input" name="' date-picker--id '-input" aria-label="Date picker"')}}
```

Comment:
Can you change `value="03/05/2020"` to `value="2020-03-05"`? We also made that change upstream.